docs/src/internal-functions/internal-modules/system-module.md

```diff
@@ -40,4 +40,11 @@ Mood today: <% tp.system.prompt("What is your mood today ?", "happy") %>
 
 Mood today: <% tp.system.suggester(["Happy", "Sad", "Confused"], ["Happy", "Sad", "Confused"]) %>
 Picked file: [[<% (await tp.system.suggester((item) => item.basename, app.vault.getMarkdownFiles())).basename %>]]
-```
\ No newline at end of file
+
+
+<%*
+const execution_value = tp.system.suggester(["Yes", "No"], ["true", "false"])
```

Comment:
You're missing an await here

Reply:
That would be problematic. Fixing **ASAP**

Reply:
Ok this is fixed.